frontend/src/plugins/auth.js

```diff
@@ -1,32 +1,32 @@
 import axios from 'axios'
-import { get, reload, post, href, purgeAll } from '@/plugins/store/apiPlugin'
+import { apiStore } from '@/plugins/store'
 import router from '@/router'
 
 axios.interceptors.response.use(null, error => {
   if (error.status === 401) {
-    logout()
+    logout().then(() => {})
```

Comment:
wozu ist das gut? 
oder sind es debug-überbleibsel?

Reply:
PHPStorm hat sich beklagt dass der zurückgegebene Promise von `logout()` ignoriert wird. Ist aber aus Sicht von ESLint okay, vielleicht weil ESLint nicht wissen kann dass `logout()` einen Promise zurückgibt. Kann es auch wieder entfernen.

Reply:
ich verstehe...

Es hat nichts mit dem PR zu tun - aber dennoch... 
Wieso braucht es hier überhaupt ein 'logout()'? Der Server hat ja soeben mit 401 (Unauthorized) gemeldet.

Reply:
Das `logout()` ist sehr wichtig um das Frontend in diesem unerwarteten Zustand wieder in Einklang mit dem Status im Backend zu bringen. `logout()` ist ja weiter unten definiert:
* Ruft `logout` Endpoint im Backend auf
* Erneuert den gecacheten Login-Zustand im Vuex store
* Leitet den User auf die Login-Seite weiter
* Löscht sämtliche Inhalte im Vuex-Store, die ein nicht eingeloggter User nicht wissen darf

Reply:
stimmt 👍